recipe/build.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+# Get c++20 to work
+if [[ $(uname) == "Darwin" ]]; then
+  export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
```

Comment:
soooo this means that macos does not support things like `#include <version>` anymore? Or what is this flag used for?

Reply:
> https://conda-forge.org/docs/maintainer/knowledge_base/#newer-c-features-with-old-sdk

Oh... please add that link as a comment